docs/components/compact.md

```diff
@@ -47,6 +47,8 @@ There's also a case when you might want to disable downsampling at all with `deb
 
 Ideally, you will have equal retention set (or no retention at all) to all resolutions which allow both "zoom in" capabilities as well as performant long ranges queries. Since object storages are usually quite cheap, storage size might not matter that much, unless your goal with thanos is somewhat very specific and you know exactly what you're doing.
 
+Setting no retention, i.e. `--retention.resolution-X=0d`, will mean that samples at the `X` resolution level will be kept forever.
```

Comment:
I will also mention default behaviour if you do not specify this flag at all (: 

```suggestion
Not setting this flag or setting it to `0d`, i.e. `--retention.resolution-X=0d`, will mean that samples at the `X` resolution level will be kept forever.
```

Reply:
Great point - I've included this in my latest commit.

Reply:
Are you sure? GH's UI still shows the old text.

Reply:
Oh whoops! I just forgot to push it 😶 it's there now :)